### ISSUE TITLE
Hot reload does not update the page

### ISSUE THREAD

**Issue by edhager** [dojo/cli-build-app#186]
Steps to reproduce:

1. Download the demo code for the [Your first Dojo application](https://dojo.io/tutorials/001_static_content/) tutorial: https://dojo.io/tutorials/assets/001_static_content-initial.zip
1. Unzip the file, and run `npm install` in the `001_static_content/initial/biz-e-corp` directory.
1. Run the Dojo build command `dojo build --mode dev --watch memory --serve`.
1. Open `http://localhost:9999` in a browser.
1. Make changes to the `001_static_content/initial/biz-e-corp/src/index.html` file.

The tutorial says that as you make changes to `index.html`, the page in the browser will reload and display those changes.  I see no changes and no errors in the console.

I do see activity in the `__webpack_hmr` event stream.  I see a hot update request (`http://localhost:9999/6161080c051786155068.hot-update.json`) in the Network tab of the Chrome dev tools that returns JSON like this:
`{"h":"4379e09eddf388653732","c":{}}`.  And I see console logging like this:
```
[HMR] connected
[HMR] bundle rebuilding
[HMR] bundle rebuilt in 744ms
[HMR] Checking for updates on the server...
[HMR] Nothing hot updated.
[HMR] App is up to date.
```

**Comment by edhager** [comment]
I used the `dojo create` command to create a new Dojo application and then I saw the error described in #183.  I tried my scenario above with the fix in pull request #184.  While the PR did fix the error described in #183 with an application I created with `dojo create`, it did not fix the problem described above with the tutorial sample code.

**Comment by KaneFreeman** [comment]
The issue you are seeing with the index.html is not the same as that fixed in #184 . There is an open issue on the webpack-dev-server repo describing the same issue you are seeing: https://github.com/webpack/webpack-dev-server/issues/1271

And another one in the webpack-hot-middleware itself that has a potential solution: https://github.com/webpack-contrib/webpack-hot-middleware/issues/237

**Comment by agubler** [comment]
This is fixed in the latest cli-build-app releases
